portal/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.server.Directives.{complete, get, onComplete, parameter, path}
+import cromwell.pipeline.datastorage.dto.user.UserResponse
+import play.api.libs.json.Json
+import scala.concurrent.Future
+import scala.util.Failure
+import cromwell.pipeline.service.{UserService}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import scala.concurrent.ExecutionContext
+import scala.util.Success
+
+class UserController (userService: UserService)(implicit executionContext: ExecutionContext) {
+  val route: Route =
+    path("users") {
+      get {
+        parameter('email.as[String]) { email =>
+          val res: Future[Seq[UserResponse]] = userService.getUsersByEmail(email)
+          onComplete(res) {
+            case Success(r)                            => complete(getSeqJson(r))
+            case Failure(exception)                    => complete(StatusCodes.InternalServerError)
```

Comment:
No need to give a name for a variable if you are not using it. You can use `_`

Reply:
have done